crates/stackable-versioned-macros/src/lib.rs

```diff
@@ -172,39 +388,72 @@ mod consts;
 /// pub struct Foo {
 ///     #[versioned(
 ///         added(since = "v1beta1"),
-///         deprecated(since = "v1", note = "not needed")
+///         deprecated(since = "v1")
 ///     )]
 ///     deprecated_bar: usize,
 ///     baz: bool,
 /// }
 /// ```
 ///
-/// #### Customize Default Function for Added Fields
+/// ### Skipping at the Version Level
 ///
-/// It is possible to customize the default function used in the generated
-/// [`From`] implementation for populating added fields. By default,
-/// [`Default::default()`] is used.
+/// Disabling this feature at the version level results in no `From`
+/// implementation for that particular version. This can be read as "skip
+/// generation for converting _this_ version to the next one". In the example
+/// below no conversion between version `v1beta1` and `v1` is generated.
```

Comment:
```suggestion
/// Disabling this behavior at the version level results in no `From`
/// implementation for that particular version. This can be read as "skip
/// generation for converting _this_ version to the next one". In the example
/// below no conversion between version `v1beta1` and `v1` is generated.
```